src/actions/renderActions.js

```diff
@@ -1,16 +1,15 @@
-export const RENDER_ORGS = 'RENDER_ORGS';
 export const RENDER_LOGIN = 'RENDER_LOGIN';
 export const RENDER_REPOS = 'RENDER_REPOS';
 export const RENDER_DASHBOARD = 'RENDER_DASHBOARD';
+export const GET_USERNAME = 'GET_USERNAME';
+// const AUTH_ROUTE = 'http://localhost:8000/isAuthenticated';
 
-export const renderOrgs = userName => (dispatch) => {
-  dispatch({
-    type: RENDER_ORGS,
-    responseStr: 'orgs',
+export const renderOrgs = (userName) => (dispatch) => {
```

Comment:
I don't think this is a good name for this action creater, it actually doesn't render anything (which it shouldn't, that's not the job of action creators)